unified-codes/apps/web/src/components/containers/detail/DetailEntityListItem.tsx

```diff
@@ -63,7 +63,7 @@ const DetailEntityListItem: DetailEntityListItem = ({ description, childEntities
     if (!childCount) return null;
     const childItems = childEntities?.map((child: IEntity) => {
       const { description, children } = child;
-      return <DetailEntityListItem description={description} childEntities={children} />;
+      return <DetailEntityListItem description={description} childEntities={children} key={child.description} />;
```

Comment:
Suggest using `code` for the key, since it's unique?

Reply:
```suggestion
      const { code, description, children } = child;
      return <DetailEntityListItem key={code} description={description} childEntities={children} />;
```

Reply:
I agree with you that using description is icky but unfortunately, code doesn't work at the moment as there are currently many intermediary nodes that are `null` which results in duplicate key errors. 

I think we either adjust this to use code with #171 or expose the underlying `uid` and use that (but I personally feel this is even worse as the UI shouldn't know about Dgraph uids).

Reply:
Agree with first option + that no nodes should have `null` codes.

Reply:
Sounds good to me - added a note to the issue description so we don't forget (issue now has 3 little 🐷 )

Reply:
ed1c3c5